docs/framework/data/adonet/sql/sql-server-express-user-instances.md

```diff
@@ -45,7 +45,7 @@ sp_configure 'user instances enabled','0'
   
 - The `|DataDirectory|` substitution string enclosed in the pipe symbols refers to the data directory of the application opening the connection and provides a relative path indicating the location of the .mdf and .ldf database and log files. If you want to locate these files elsewhere, you must provide the full path to the files.  
   
-```  
+```csharp
```

Comment:
I don't think this is an actual C# code. (These applies to some other similar snippets too)

Reply:
You're right. I'm changing them to "text". They are examples of SQLConnection strings that are consumed by our C# API. So, text values of a C# property, not C#. Thanks!